charts/incubator/external-dns/questions.yaml

```diff
@@ -0,0 +1,324 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+
+
+                                - variable: env
+                                  group: "App Configuration"
+                                  label: "Provider Credentials"
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
```

Comment:
Maybe we should have a dropdown for providers and only show vars fro the selected provider?
will make screenshots for support shorter

Reply:
Yep, I just gotta find a good example lol

Reply:
find an `enum` + a `show_if` example!